src/routes/PhotoRouter.ts

```diff
@@ -20,6 +21,8 @@ class PhotoRouter implements IRouter {
   initializeRoutes() {
     this.router.get(`${this.path}/:id(\\d+)`, this.photoController.getById);
     this.router.post(this.path, myValidator(PhotoCreateValidator, true), this.photoController.create);
+    this.router.put(`${this.path}/:id(\\d+)`, myValidator(PhotoUpdateValidator, true), this.photoController.updateById);
```

Comment:
Consider using `PATCH` in this case. We should use `PUT` only when we expect to receive a complete entity (i.e., with all its required fields) that completely replaces what we already have. We should use `PATCH` when we want to apply a partial update to an existing entity. 

```suggestion
    this.router.patch(`${this.path}/:id(\\d+)`, myValidator(PhotoUpdateValidator, true), this.photoController.updateById);
```

> TL;DR [An interesting discussion about this in SO](https://stackoverflow.com/questions/28459418/use-of-put-vs-patch-methods-in-rest-api-real-life-scenarios).